tests/integration/docker_utils/test_docker.py

```diff
@@ -1432,6 +1433,28 @@ def test_run_with_additional_arguments_random_port(
         )
         assert automatic_host_port > 0
 
+    def test_run_with_ulimit(self, docker_client: ContainerClient):
+        # test baseline without ulimit
+        container_name = f"c-{short_uid()}"
+        stdout, _ = docker_client.run_container(
+            "alpine",
+            name=container_name,
+            remove=True,
+            command=["sh", "-c", "ulimit -n"],
+        )
+        assert stdout.decode(config.DEFAULT_ENCODING).strip() == "1073741816"
```

Comment:
This assertion will be highly system specific. I would rather avoid constraints which make the test fail depending on the system limit configuration.

Reply:
Is it? I thought those limit were set by the image. I will remove that section.

Reply:
Usually, those are inherited from the docker daemon, and are whatever that process is launched with :)